tasklist/config/docker-compose.identity.yml

```diff
@@ -187,7 +187,7 @@ services:
       elasticsearch:
         condition: service_started
     restart: on-failure
-    image: camunda/tasklist:8.5.1
+    image: camunda/tasklist:local-hb
```

Comment:
@houssain-barouni Should this be left like this?

Reply:
no, it is a local docker image, I will fix it